### DISCUSSION TITLE
How can I translate the chat box

### DISCUSSION THREAD

**Discussion by le-show-de-didoub74** [dimaguy/ImpostorHQ#27] [Q&A] [answered]
Hello, how did I do

For exemple if I do `/help` I don't want the result in english

**Comment by dimaguy** [comment, marked as answer]
The embedded command results are hardcoded in the software, you'll need to manually edit the source to translate it over to your language

**Reply by le-show-de-didoub74** [reply]
Okay The different command are in what plugin ?

**Reply by dimaguy** [reply]
Some commands are hard coded onto the core, but others are introduced via different plugins. /help will list the existing commands

**Reply by le-show-de-didoub74** [reply]
And you use Visual Studio for coding ?

**Reply by dimaguy** [reply]
Yes

**Reply by le-show-de-didoub74** [reply]
Okay thanks
